### DISCUSSION TITLE
Welcome to MetaTermKotlin Discussions!

### DISCUSSION THREAD

**Discussion by diegoberaldin** [diegoberaldin/MetaTerm#15] [Announcements]
## 🙇‍♂️ Welcome!  Thank you for being here 🙏

This space is intended to connect with other members of our community. We hope that you:
  * Ask questions you're wondering about
  * Share your ideas/suggestions/criticism
  * Engage with other community members
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by deutschluo** [comment]
Hi diegoberaldin,
I am a translator and is looking for an OpenSource terminology management tool. I was delighted when I found the MetaTerm. 
But it is not very clear for me how to configurate a Termbase under Windows system. I have selected 3 languages. And how to configurat the "definition model"? What is the meaning of name, level and type?
Futhermore, I have selected all properties of the "Input model". After clicking OK the termbase haven't been created successfully. 
Is it possible to make a tutorial for creating termbase? I would appreciate it.
![1](https://github.com/diegoberaldin/MetaTerm/assets/156651040/ac713913-db12-48e5-9391-e8c986ea165d)

Best regards
Yang

**Reply by deutschluo** [reply]
Besides, would it be possible to add Chinese into the "Languages"?

**Reply by diegoberaldin** [reply]
Hi! This project has never had a real user and I considered it "dead", let's see if it can be resumed... I took inspiration from MultiTerm by old Trados (now SDL) for those concepts:
- the definition model is the structure of terminological entries, i.e. the set of properties that each entry can have, the type of each property and whether the property applies to individual terms, to all the terms of a language or to the whole terminological entry;
- the input model defines which properties, among the whole group of the definition model, are shown by default while inserting a new entry; the hidden ones are just there and can be accessed using the "Add property" button (for term level, language level or entry level) in the entry creation form.

As for adding Chinese, of course it is possible and sorry for not doing it already, I just randomly added all the languages of the EU but without having any user I had no clue about what could be needed.

Please keep in mind that the app is experimental, and it's open source with just 1 maintainer (me) so don't rely heavily only on it for work. I hoped the project could gain momentum last year but it never really lifted off.

**Reply by deutschluo** [reply]
I see. I have tried MultiTerm several years ago and know the principle a little. 
After I finished the setting for the termbase and clicked OK, MetaTerm was loading and loading. And it didn't end which means for me that the termbase can't be created. 
But I understand your situation. Without other developers or feedbacks, it is difficult for this project.
